src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  &__star {
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    list-style: none;
+    background-image: url("../images/star.svg");
+  }
+}
+  .stars--N .stars__star:nth-child(-n+4) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
The selector `.stars--N .stars__star:nth-child(-n+4)` is hardcoded to highlight the first 4 stars. To handle different star ratings dynamically, consider using a more flexible approach, such as dynamically setting the class based on the rating value (e.g., `.stars--3` for a 3-star rating).